osu-ac/cacher.py

```diff
@@ -34,7 +35,8 @@ def cache(map_id, user_id, lzma_string, replay_id):
         """
 
         compressed_string = Cacher.compress(lzma_string)
-        Cacher.write("INSERT INTO replays VALUES(?, ?, ?, ?)", [map_id, user_id, compressed_string, replay_id])
+        packed_osr = wtc.compress(compressed_string)
```

Comment:
return wtc.compress(lzma_string) from Cacher.compress, I left that empty specifically for this implementation